src/erlcloud_firehose_impl.erl

```diff
@@ -0,0 +1,177 @@
+%% -*- mode: erlang;erlang-indent-level: 4;indent-tabs-mode: nil -*-
```

Comment:
Would you like the firehose calls in erlcloud_kinesis or separate module as well?